internal/benchmark/CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ foreach(thrust_target IN LISTS THRUST_TARGETS)
   target_include_directories(${bench_target} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}")
   thrust_clone_target_properties(${bench_target} ${thrust_target})
 
+  if ("CUDA" STREQUAL "${config_device}")
+    thrust_set_rdc_state(${example_target} OFF)
```

Comment:
no benchmark version of `THRUST_ENABLE_TESTS_WITH_RDC`?

Reply:
Good question -- nope, this is a legacy benchmark that is planned to be replaced by the NVBench suite. The build system for it is pretty barebones and just emulates old behaviors.